rats-tls/dist/rpm/rats-tls-tdx-filelist

```diff
@@ -0,0 +1,8 @@
+/usr/share/rats-tls/samples/rats-tls-server
+/usr/share/rats-tls/samples/rats-tls-client
+/usr/local/include/rats-tls/rats-tls/*.h
+/usr/local/lib/rats-tls/librats_tls.so*
+/usr/local/lib/rats-tls/tls-wrappers/libtls_wrapper*.so*
+/usr/local/lib/rats-tls/crypto-wrappers/libcrypto_wrapper*.so*
+/usr/local/lib/rats-tls/attesters/libattester*.so*
+/usr/local/lib/rats-tls/verifiers/libverifier*.so*
```

Comment:
These three files `rats-tls-tdx-filelist`,`rats-tls-host-filelist`,`rats-tls-occlum-filelist`  have the same content. Please use one file to reduce duplicate